### PR TITLE
Refetch balance on deposit

### PR DESCRIPTION
**Pull request by Tulun**
Balance below AddLiquidity was incorrect after transaction. Refetch the balances after successful deposit / withdrawal

### CODE REVIEW

File: src/views/Pool.tsx [hunk 1]
```diff
@@ -66,6 +66,15 @@ const Pool: FC = () => {
     }
   }, [isConnected, connection.account, token.bridgePool]);
 
+  useEffect(() => {
```
Comment (daywiss) [inline src/views/Pool.tsx:69]: why is this fetched twice? its already being done in the remove liquidity component

File: src/components/PoolForm/AddLiquidityForm.tsx [hunk 2]
```diff
@@ -14,10 +14,11 @@ import {
 import { poolClient } from "state/poolsApi";
 import { toWeiSafe } from "utils/weiMath";
 import { useERC20 } from "hooks";
-import { ethers, BigNumber } from "ethers";
+import { ethers } from "ethers";
 import { clients } from "@uma/sdk";
 import { addEtherscan } from "utils/notify";
 import BouncingDotsLoader from "components/BouncingDotsLoader";
+import { ADD_LIQUIDITY_ETH_GAS, GAS_PRICE_BUFFER } from "utils/constants";
```
Comment (daywiss) [inline src/components/PoolForm/AddLiquidityForm.tsx:21]: is this needed?

Comment (Tulun) [reply src/components/PoolForm/AddLiquidityForm.tsx:21]: Was throwing a compile error.

File: src/components/PoolForm/RemoveLiquidityForm.tsx [hunk 3]
```diff
@@ -105,6 +108,10 @@ const RemoveLiqudityForm: FC<Props> = ({
             const url = `https://etherscan.io/tx/${transaction.hash}`;
             setShowSuccess(true);
             setDepositUrl(url);
+            // Recheck for balances. note: Onboard provider is faster than ours.
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/components/PoolForm/RemoveLiquidityForm.tsx:111]: yes

File: src/components/PoolForm/AddLiquidityForm.tsx (hunk 2 above)
Comment (daywiss) [reply src/components/PoolForm/AddLiquidityForm.tsx:21]: i see yea its needed in this component, somehow it passed a pr withit out

**Review by daywiss — APPROVED** [review]
looks good just remove unused vars

**Pull request merged**